services/inspection-report/src/views/Inspections/InspectionList/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { FixedSizeList } from 'react-window';
+import InfiniteLoader from 'react-window-infinite-loader';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import './styles.css';
+import { Loader } from '../../../components';
+
+export default function InspectionList({ items, loadMore, hasNextPage }) {
+  // eslint-disable-next-line react/prop-types,react/no-unstable-nested-components
+  function Item({ index, style }) {
+    const navigate = useNavigate();
+    const navigateToDetails = () => {
+      if (index < items.length) {
+        navigate(`/inspections/${items[index].id}`);
+      }
+    };
```

Comment:
This can be defined outside the `InspectionList` component or use a callback like:
```js
const renderItem = useCallback(({ index, ...passThoughProps }) => {
  const item = items[index];

  return <ListItem {...item} {...passThoughProps} />;
}, [items])
```